openssl/src/stack.rs

```diff
@@ -72,12 +71,6 @@ impl<T: Stackable> AsRef<StackRef<T>> for Stack<T> {
     }
 }
 
-impl<T: Stackable> Borrow<StackRef<T>> for Stack<T> {
```

Comment:
`Stack<T>` doesn't have a `foreign_type!`-generated impl, so isn't this still necessary?

Reply:
Oops, restored.